content/en/guides/features/deployment-targets.md

```diff
@@ -7,31 +7,62 @@ category: features
 
 ## Static Hosting
 
-For static hosting (hosting where no server is needed) the target of static needs to be added to your nuxt.config file.
+With static hosting, hosting where no server is needed, you can choose to either host a single page application or a multiple page application, also known as static sties. With static hosting no server is needed, meaning your SPA or static site can be hosted on any serverless hosting or CDN for free. For static hosting the target of `static` needs to be added to your `nuxt.config` file.
```

Comment:
I think would be better to use `any static hosting` instead of `any serverless hosting` because serverless target is server  + not always is free.